src/Core/WinSWCore/Download.cs

```diff
@@ -29,16 +29,21 @@ public enum AuthType
 
         private static readonly ILog Logger = LogManager.GetLogger(typeof(Download));
 
-        public readonly string From;
-        public readonly string To;
-        public readonly AuthType Auth;
-        public readonly string? Username;
-        public readonly string? Password;
-        public readonly bool UnsecureAuth;
-        public readonly bool FailOnError;
-        public readonly string? Proxy;
+        public string from;
+        public string to;
+        public AuthType auth;

```

Comment:
Should it default to AuthType.none and be optional?

Reply:
This always defaults to `none`.